src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -16,24 +17,50 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     @track visualforceOrigin;
     @track isLoading = true;
     @track alert = {};
+    @track disabledMessage;
+    @track isDisabled = false;
 
     CUSTOM_LABELS = GeLabelService.CUSTOM_LABELS;
     tokenizeCardPageUrl = '/apex/GE_TokenizeCard';
 
-
-    get tokenizeCardHeader() {
-        return GeLabelService.format(
-            this.CUSTOM_LABELS.geHeaderPaymentServices,
-            [this.CUSTOM_LABELS.commonPaymentServices]);
-    }
-
     async connectedCallback() {
         this.domain = await getOrgDomain();
         this.visualforceOrigin = this.buildVisualforceOriginUrl(this.domain);
     }
 
     renderedCallback() {
         this.registerPostMessageListener();
+        registerListener('disableGeFormWidgetTokenizeCard', this.handleDisableWidget, this);
```

Comment:
Can we move this event name into geConstants.js and import it wherever used? I don't think we need to do this for every event, only ones that are fired from one component and handled in another.

Reply:
Updated!